cpp/oneapi/dal/algo/rbf_kernel/backend/cpu/compute_kernel_test.cpp

```diff
@@ -0,0 +1,178 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "gtest/gtest.h"
+#include "oneapi/dal/algo/rbf_kernel.hpp"
+#include "oneapi/dal/data/accessor.hpp"
+#include "oneapi/dal/data/table.hpp"
+
+#include <cmath>
+
+using namespace oneapi::dal;
+using std::int32_t;
+
+TEST(rbf_kernel_dense_test, can_compute_unit_matrix) {
+    constexpr std::int64_t row_count    = 2;
+    constexpr std::int64_t column_count = 2;
+    const float x_data[]                = {
+        1.f,
+        1.f,
+        1.f,
+        1.f,
+    };
+    const float y_data[] = {
+        1.f,
+        1.f,
+        1.f,
+        1.f,
+    };
+
+    const auto x_table = homogen_table{ row_count, column_count, x_data };
+    const auto y_table = homogen_table{ row_count, column_count, y_data };
+
+    const auto kernel_desc  = rbf_kernel::descriptor{};
+    const auto values_table = compute(kernel_desc, x_table, y_table).get_values();
+    ASSERT_EQ(values_table.get_row_count(), row_count);
+    ASSERT_EQ(values_table.get_column_count(), row_count);
+
+    auto values_accessor   = row_accessor<const float>(values_table).pull();
+    const auto values_data = values_accessor.get_data();
+
+    for (size_t i = 0; i < values_accessor.get_count(); i++) {
+        ASSERT_FLOAT_EQ(values_data[i], 1.f);
+    }
```

Comment:
You can avoid getting raw pointer
```suggestion
    for (size_t i = 0; i < values_accessor.get_count(); i++) {
        ASSERT_FLOAT_EQ(values_accesor[i], 1.f);
    }
```